backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/services/EMFValidationService.java

```diff
@@ -64,7 +71,19 @@ public EMFValidationService(EValidator.Registry eValidatorRegistry) {
 
         Map<Object, Object> options = new HashMap<>();
         options.put(Diagnostician.VALIDATE_RECURSIVELY, true);
-        Diagnostician diagnostician = new Diagnostician(this.eValidatorRegistry);
+        Diagnostician diagnostician = new Diagnostician(this.eValidatorRegistry) {
+            @Override
+            public String getObjectLabel(EObject eObject) {
+                if (EMFValidationService.this.composedAdapterFactory != null) {
```

Comment:
Is this needed? It's `final` and initialized to a necessarily non-null value in the constructor.

Reply:
Indeed, I should check that composedAdapterFactory is an instance of IItemLabelProvider